ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.js

```diff
@@ -122,17 +134,129 @@
       },
     },
     {
-      id: 'Fractal Death Spin',
-      netRegex: NetRegexes.startsUsing({ id: '27AE', source: 'The Ultima Beast', capture: false }),
-      netRegexDe: NetRegexes.startsUsing({ id: '27AE', source: 'Ultimativ(?:e|er|es|en) Bestie', capture: false }),
-      netRegexFr: NetRegexes.startsUsing({ id: '27AE', source: 'Ultima-Monstre', capture: false }),
-      netRegexJa: NetRegexes.startsUsing({ id: '27AE', source: 'アルテマビースト', capture: false }),
-      netRegexCn: NetRegexes.startsUsing({ id: '27AE', source: '究极神兽', capture: false }),
-      netRegexKo: NetRegexes.startsUsing({ id: '27AE', source: '알테마 비스트', capture: false }),
+      id: 'Fractal Hard Ram Voice Proto',
+      netRegex: NetRegexes.startsUsing({ id: '860', source: 'Proto-Chimera', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '860', source: 'Proto-Chimära', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '860', source: 'protochimère', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '860', source: 'プロトキマイラ', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '860', source: '原型奇美拉', capture: false }),
+      netRegexKo: NetRegexes.startsUsing({ id: '860', source: '프로토 키마이라', capture: false }),
+      alertText: {
+        en: 'Ram\'s Voice',
+        de: 'Stimme Des Widders',
+        fr: 'Voix Du Bélier',
+        ja: '氷結の咆哮',
+        cn: '远离奇美拉',
+        ko: '빙결의 포효',
+      },
+      tts: {
```

Comment:
Is there a reason to have a special `tts` here?

Reply:
I have no idea, the trigger in its entirety was already present. I just did a bit of copypasta for both forms of Chimera. (Line 101/113 is the original. I've actually never seen the Servomechanical Chimera use Ram's Voice, but I didn't want to just remove it entirely, since there was a specific trigger for it. The same goes for the Proto-Chimera and Dragon's Voice.)

Reply:
Ah ok.  I think in these's cases, we could probably just remove the tts.